element_interface/intan_loader.py

```diff
@@ -248,40 +240,48 @@ def load_rhs(folder: str, file_expr: str):
             rhs_data["timestamps"] (np.array_like): Relative timestamps in seconds.
     """
 
+    rhs_data = {}
+
+    # Get header
     header_filepath = next(Path(folder).glob("info.rhs"))
     with open(header_filepath, "rb") as fid:
-        header = read_header(fid)
+        rhs_data["header"] = _read_header(fid)
 
+    # Get timestamps
     time_file = next(Path(folder).glob("time.dat"))
 
-    timestamps = (
+    rhs_data["timestamps"] = (
         np.memmap(time_file, dtype=np.int32)
-        / header["frequency_parameters"]["amplifier_sample_rate"]
+        / rhs_data["header"]["frequency_parameters"]["amplifier_sample_rate"]
     )
 
-    rhs_data = dict(header=header, timestamps=timestamps, recordings={})
-
+    # Get data files
     file_paths = Path(folder).glob(file_expr)
-    file_paths = [x for x in file_paths if x.as_posix() != "time.dat"]
 
-    for file_path in file_paths:
-        file_path = file_path.as_posix()
-        if "amp" in file_path:
+    if file_expr == "*":
+        file_paths = list(folder.glob("[!info|!time|!Zone]*"))
+
+    # Get recording data
+    rhs_data["recordings"] = {}
+
+    for file_path in sorted(file_paths):
+        signal_type = file_path.stem.split("-")[0]
+
+        if signal_type == "amp":
             signal = np.memmap(file_path, dtype=np.int16)
             signal = signal * 0.195  # Convert to microvolts
-        elif "board-ANALOG-IN" in file_path or "board-ANALOG-OUT" in file_path:
-            signal = np.memmap(file_path, dtype=np.uint16)
-            signal = (signal - 32768) * 0.0003125  # Convert to volts
-        elif "dc-" in file_path:
-            signal = np.memmap(file_path, dtype=np.uint16)
-            signal = (signal - 512) * 19.23  # Convert to milivolts
-        elif "board-DIGITAL-IN" in file_path or "board-DIGITAL-OUT" in file_path:
+        else:
             signal = np.memmap(file_path, dtype=np.uint16)
-        elif "stim-" in file_path:
-            data = np.memmap(file_path, dtype=np.uint16)
-            i = np.bitwise_and(data, 255) * header["stim_step_size"]
-            sign = (128 - np.bitwise_and(data, 255)) / 128
-            signal = i * sign
-        rhs_data["recordings"][Path(file_path).relative_to(folder).stem] = signal
+            if signal_type == "board":
```

Comment:
The original version was more readable. We were able to see the datatype and conversions applied to reading from each file.  Now there is an if-elif after and elif.